qiskit/algorithms/minimum_eigensolvers/minimum_eigensolver.py

```diff
@@ -72,9 +73,19 @@ class MinimumEigensolverResult(AlgorithmResult):
 
     def __init__(self) -> None:
         super().__init__()
+        self._ansatz = None
         self._eigenvalue = None
         self._aux_operators_evaluated = None
 
+    @property
+    def ansatz(self) -> QuantumCircuit:
+        """The ansatz to retrieve the minimum eigenstate."""
```

Comment:
How about the following? 🙂 
```suggestion
        """The circuit preparing the parameterized ansatz state.
        
         Along with the optimal parameters, this can be used to retrieve the minimum eigenstate.
         """
```